README.md

```diff
@@ -1,25 +1,25 @@
-# Ignition Plugin
+# Gazebo Plugin
 
 **Maintainer:** grey [AT] openrobotics [DOT] org
 
-[![GitHub open issues](https://img.shields.io/github/issues-raw/ignitionrobotics/ign-plugin.svg)](https://github.com/ignitionrobotics/ign-plugin/issues)
-[![GitHub open pull requests](https://img.shields.io/github/issues-pr-raw/ignitionrobotics/ign-plugin.svg)](https://github.com/ignitionrobotics/ign-plugin/pulls)
+[![GitHub open issues](https://img.shields.io/github/issues-raw/gazebosim/gz-plugin.svg)](https://github.com/gazebosim/gz-plugin/issues)
+[![GitHub open pull requests](https://img.shields.io/github/issues-pr-raw/gazebosim/gz-plugin.svg)](https://github.com/gazebosim/gz-plugin/pulls)
 [![Discourse topics](https://img.shields.io/discourse/https/community.gazebosim.org/topics.svg)](https://community.gazebosim.org)
 [![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-plugin/branch/ign-plugin1/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-plugin/branch/ign-plugin1)
-Ubuntu Bionic | [![Build Status](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-bionic-amd64/badge/icon)](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-bionic-amd64/)
+Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-plugin/branch/ign-plugin1/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-plugin/branch/ign-plugin1)
+Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-focal-amd64/badge/icon)](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-focal-amd64/)
 Homebrew      | [![Build Status](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-bionic-amd64/badge/icon)](https://build.osrfoundation.org/job/ignition_plugin-ci-ign-plugin1-bionic-amd64/)
```

Comment:
I just noticed that the homebrew badge is pointing to Bionic 🤦🏽‍♀️ Do you mind fixing it?

Reply:
Good catch! 35032d1e98d493478d32a72548d720701051230a